app/controllers/contacts_controller.rb

```diff
@@ -5,7 +5,7 @@ class ContactsController < ApplicationController
   before_action :set_contact_type, only: %i[update create]
 
   def index
-    @contacts = Contact.includes([:contact_type]).all.decorate
+    @contacts = Contact.order(name: :asc).includes([:contact_type]).all.decorate
```

Comment:
Looks good! I think the `all` here is redundant (and already was) so you could remove that.
Also, while there is no rule around this, people tend to put order clauses at the end of a query like this. It doesn't make any difference to what it does, it's just a style thing. 
i.e. `@contacts = Contact.includes([:contact_type]).order(name: :asc).decorate`

I can see that there isn't a spec for this controller for some reason, but it would be good to at least add one with a test for this change.

Reply:
Thank you - I will amend the change I have made and also add a spec and test